code/modules/paperwork/paper.dm

```diff
@@ -224,7 +231,7 @@
 		get_asset_datum(/datum/asset/spritesheet/simple/paper),
 	)
 
-/obj/item/paper/ui_interact(mob/user, datum/tgui/ui)
+/obj/item/paper/ui_interact(mob/user, datum/tgui/ui=null)
```

Comment:
redundant initialization to null

Reply:
Yea I was trying to figure out why it wasn't working on the door...waaait...maybe its because its in the door contests.

Reply:
Sleep helps